src/diffpy/pdfmorph/morphs/morphshift.py

```diff
@@ -47,10 +47,19 @@ class MorphShift(Morph):
 
     def morph(self, x_morph, y_morph, x_target, y_target):
         """Apply the shifts."""
+        try:
+            hshift = self.hshift
+        except AttributeError:
+            hshift = 0
+        try:
+            vshift = self.vshift
+        except AttributeError:
+            vshift = 0
+
```

Comment:
This code ensures that the vertical/horizontal shifts can be done independent of each other. We can also create two classes: MorphHShift and MorphVShift, but I feel that is overkill.